cgs_vmc/utils.py

```diff
@@ -107,6 +107,9 @@ def create_hparams(**kwargs: Any) -> tf.contrib.training.HParams:
       bot_lin_table_file='',
       ed_vector_file='',
 
+      # GraphConvNetwork parameters
+      adjacency_list = '',
```

Comment:
As per style guide [link](http://google.github.io/styleguide/pyguide.html#36-whitespace) whitespace is not used for default values unless type annotation is used.

`adjacency_list='',`